frontend/src/metabase/query_builder/components/expressions/Editor/custom-tooltip.tsx

```diff
@@ -0,0 +1,131 @@
+import {
```

Comment:
Is this expected?

![image](https://github.com/user-attachments/assets/29cf5d88-f872-4c24-8d29-c6dea45a535d)